tests/vcs/git/GitCommitBearTest.py

```diff
@@ -288,6 +288,22 @@ def test_body_checks(self):
                              body_regex=r'TICKER\s*CLOSE\s+[1-9][0-9]*'), [])
         self.assert_no_msgs()
 
+        # Testing if check recognises invalid emails
+        self.git_commit('Shortlog\n\n'
+                        'A valid email rhemon19@gmail.com\n'
+                        'An invalid email invalidemail@invalid.invalid\n'
```

Comment:
Line is longer than allowed. (87 > 79)

*LineLengthBear, severity NORMAL, section `linelength`.*